test/unit/specs/components/staking/UndelegationModal.spec.js

```diff
@@ -49,76 +58,75 @@ describe(`UndelegationModal`, () => {
     it(`account password defaults to an empty string`, () => {
       expect(wrapper.vm.password).toEqual(``)
     })
-
-    it(`password is hidden by default`, () => {
-      expect(wrapper.vm.showPassword).toBe(false)
-    })
-  })
-
-  describe(`Password display`, () => {
-    it(`toggles the password between text and password`, () => {
-      wrapper.vm.togglePassword()
-      expect(wrapper.vm.showPassword).toBe(true)
-      wrapper.vm.togglePassword()
-      expect(wrapper.vm.showPassword).toBe(false)
-    })
   })
 
-  describe(`enables or disables the Delegation button correctly`, () => {
-    describe(`disables the 'Delegation' button`, () => {
+  describe(`only submits on correct form`, () => {
+    describe(`does not submit`, () => {
       it(`with default values`, () => {
-        let undelegationBtn = wrapper.find(`#submit-undelegation`)
-        expect(undelegationBtn.html()).toContain(`disabled="disabled"`)
+        wrapper.vm.submitForm = jest.fn()
+        wrapper.vm.validateForm()
+        expect(wrapper.vm.submitForm).not.toHaveBeenCalled()
       })
 
-      it(`if the user manually inputs a number greater than the balance`, () => {
+      xit(`if the user manually inputs a number greater than the balance`, async () => {
```

Comment:
Is this on pourpose `xit` or leftover skipping

Reply:
removed, tmfield doesn't limit to max anymore. the form should rather show an error.